bayesian/api_v1.py

```diff
@@ -425,26 +425,11 @@ class UserIntent(ResourceWithSchema):
 
     @staticmethod
     def get(user, ecosystem):
-        if not user:
-            raise HTTPError(400, error="Expected user name in the request")
-
-        if not ecosystem:
-            raise HTTPError(400, error="Expected ecosystem in the request")
-
-        s3 = StoragePool.get_connected_storage('S3ManualTagging')
-        # get user data
-        try:
-            result = s3.fetch_user_data(user, ecosystem)
-        except botocore.exceptions.ClientError:
-            err_msg = "Failed to fetch data for the user {u}, ecosystem {e}".format(u=user,
-                                                                                    e=ecosystem)
-            current_app.logger.exception(err_msg)
-            raise HTTPError(404, error=err_msg)
-
-        return result
+        raise HTTPError(404, "Unsupported API endpoint")
```

Comment:
405 would be probably more appropriate here. But this should be transparently handled by flask (webserver? not sure). So we can remove the `get()` method altogether.